lib/foreman_templates/engine.rb

```diff
@@ -10,6 +10,18 @@ module ForemanTemplates
   class Engine < ::Rails::Engine
     engine_name 'foreman_templates'
 
+
+
+    def self.assets_to_precompile
+      assets = Dir.chdir(root) do
+        Dir['app/assets/javascripts/**/*'].map do |f|
+          f.split(File::SEPARATOR, 4).last
+        end
+      end
+    end
```

Comment:
Do we need this? Shouldn't all the same files be picked already by https://github.com/theforeman/foreman/blob/b49b89ad425c09ee35a9665ceff337ad3fcb96ad/app/registries/foreman/plugin/assets.rb#L45